### PR TITLE
Added missing deploy settings in cloud section

### PR DESCRIPTION
**Pull request by jonat123**

### CODE REVIEW

File: Umbraco-Cloud/Deployment/Deploy-Settings/index.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ verified-against: beta-1
 
 # Configuration for Umbraco Deploy
 
-All configuration for Umbraco Deploy is held in the `appSettings.json` file found at the root of your Umbraco website.  If the configuration has been customized to use another source, then the same keys and values discussed in this article can be applied there.
+All configuration for Umbraco Forms is held in the `appSettings.json` file found at the root of your Umbraco website.  If the configuration has been customized to use another source, then the same keys and values discussed in this article can be applied there.
```
Comment (eshanrnh) [inline Umbraco-Cloud/Deployment/Deploy-Settings/index.md:11]:
```suggestion
All configuration for Umbraco Deploy is held in the `appSettings.json` file found at the root of your Umbraco website.  If the configuration has been customized to use another source, then the same keys and values discussed in this article can be applied there.
```
Section heading says Deploy

File: Umbraco-Cloud/Deployment/Deploy-Settings/index.md [hunk 2]
```diff
@@ -126,6 +131,21 @@ In order for Deploy to handle Forms data as content, you'll to ensure the `Trans
 
 In a similar way, Deploy can be configured to allow for backoffice transfers of dictionary items instead of using files serialized to disk, by setting `TransferDictionaryAsContent` as `true`.
 
+## AllowMembersDeploymentOperations and TransferMemberGroupsAsContent
+
+As of version 9.3.0, it's also possible to transfer members and member groups via the back-office between environments.  This is disabled by default as a deliberate decision to make use of the feature needs to be taken, as for most installations it will make sense to have member data created and managed only in production. There are obvious potential privacy concerns to consider too.  However, if being able to deploy and restore this information between environments makes sense for the specific workflow of your project, it's a supported scenario.
```
Comment (eshanrnh) [inline Umbraco-Cloud/Deployment/Deploy-Settings/index.md:136]: Try simplifying to remove all the jargon/marketing terms.

```suggestion
This is disabled by default, as for most installations, member data is created and managed _only_ in production. If you want to deploy and restore member information between environments for specific workflows of your project, you can do so in the `appsettings.json` file
```

File: Umbraco-Cloud/Deployment/Deploy-Settings/index.md [hunk 3]
```diff
@@ -139,3 +159,40 @@ When restoring or transferring content, Umbraco Deploy will make checks to ensur
...
+For example, using the following settings, you will have an installation that ignores broken dependencies when restoring from an upstream environment.  It will however still prevent deployment and report any dependency issues when attempting a transfer to an upstream environment.
+
+```json
+    "IgnoreBrokenDependencies": true,
+    "IgnoreBrokenDependenciesBehavior": "Restore",
+```
+
+## Memory cache reload
+
+Some customers have reported intermittent issues related to Umbraco's memory cache following deployments, which are resolved by a manual reload of the cache via the _Settings > Published Status > Caches_ dashboard.  If you are running into such issues and are able to accomodate a cache clear after deployment, this workaround can be automated via the following setting:
```
Comment (eshanrnh) [inline Umbraco-Cloud/Deployment/Deploy-Settings/index.md:176]:
```suggestion
If you are running into such issues and can accommodate a cache clear after deployment, this workaround can be automated via the following setting:
```

File: Umbraco-Cloud/Deployment/Deploy-Settings/index.md [hunk 4]
```diff
@@ -139,3 +159,40 @@ When restoring or transferring content, Umbraco Deploy will make checks to ensur
...
+
+```json
+    "ReloadMemoryCacheFollowingDiskReadOperation": true,
+```
+
+By upgrading to the most recent available version of the CMS major you are running, you'll be able to benefit from the latest bug fixes and optimizations in this area.  That should be your first option if encountering cache related issues. Failing that, or if a CMS upgrade is not an option, then this workaround can be considered.
```
Comment (eshanrnh) [inline Umbraco-Cloud/Deployment/Deploy-Settings/index.md:182]:
```suggestion
By upgrading to the most recent available version of the CMS major you are running, you'll be able to benefit from the latest bug fixes and optimizations in this area.  That should be your first option if encountering cache-related issues. Failing that, or if a CMS upgrade is not an option, then this workaround can be considered.
```

File: Umbraco-Cloud/Deployment/Deploy-Settings/index.md [hunk 5]
```diff
@@ -139,3 +159,40 @@ When restoring or transferring content, Umbraco Deploy will make checks to ensur
...
+
+```json
+    "AllowDomainsDeploymentOperations": "None|Culture|AbsolutePath|Hostname|All",
+```
+
+To enable this, set the configuration value as appropriate for the types of domains you want to allow:
+
+- *Culture* - the language setting for the content, defined under "Culture"
+- *AbsolutePath* - values defined under "Domains" with an absolute path, e.g. "/en"
+- *Hostname* - values defined under "Domains" with a full host name, e.g. "en.mysite.com"
```
Comment (eshanrnh) [inline Umbraco-Cloud/Deployment/Deploy-Settings/index.md:196]:
```suggestion
- *Hostname* - values defined under "Domains" with a full hostname, e.g. "en.mysite.com"
```

File: Umbraco-Cloud/Deployment/Deploy-Settings/index.md [hunk 6]
```diff
@@ -139,3 +159,40 @@ When restoring or transferring content, Umbraco Deploy will make checks to ensur
...
+
+Combinations of settings can be applied, e.g. `Hostname,AbsolutePath`.
```
Comment (eshanrnh) [inline Umbraco-Cloud/Deployment/Deploy-Settings/index.md:198]:
```suggestion
Combinations of settings can be applied, e.g. `Hostname, AbsolutePath`.
```

**Review by eshanrnh — COMMENTED** [review]
Looks good 💪Just a couple of suggestions.

File: Umbraco-Cloud/Deployment/Deploy-Settings/index.md (hunk 1 above)
Comment (jonat123) [reply Umbraco-Cloud/Deployment/Deploy-Settings/index.md:11]: thanks did not see that before 😅

File: Umbraco-Cloud/Deployment/Deploy-Settings/index.md [hunk 7]
```diff
@@ -126,6 +131,21 @@ In order for Deploy to handle Forms data as content, you'll to ensure the `Trans
 
 In a similar way, Deploy can be configured to allow for backoffice transfers of dictionary items instead of using files serialized to disk, by setting `TransferDictionaryAsContent` as `true`.
 
+## AllowMembersDeploymentOperations and TransferMemberGroupsAsContent
+
+As of version 9.3.0, it's also possible to transfer members and member groups via the back-office between environments.  This is disabled by default as a deliberate decision to make use of the feature needs to be taken, as for most installations it will make sense to have member data created and managed only in production. There are obvious potential privacy concerns to consider too.  However, if being able to deploy and restore this information between environments makes sense for the specific workflow of your project, it's a supported scenario.
+
+To enable, you can add or amend the `AllowMembersDeploymentOperations` and `TransferMemberGroupsAsContent` settings.
+
```
Comment (eshanrnh) [inline Umbraco-Cloud/Deployment/Deploy-Settings/index.md:139]: How about adding the code snippet here? https://umbraco.com/blog/release-candidates-umbraco-forms-81193-and-deploy-4593/

**Review by eshanrnh — COMMENTED** [review]
Just one more suggestion about adding a code snippet.

**Comment by jonat123** [comment]
Closing this PR for now as we will do something differently.

**Pull request closed without merging**